table/table_builder.h

```diff
@@ -54,15 +54,16 @@ struct TableBuilderOptions {
       CompressionType _compression_type,
```

Comment:
'explicit' keyword for constructor is useful only when parameter number can be 1.